kafka/pom.xml

```
@@ -0,0 +1,177 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.stargate</groupId>
+    <artifactId>stargate</artifactId>
+    <version>0.0.11-SNAPSHOT</version>
+  </parent>
+  <groupId>io.stargate.producer.kafka</groupId>
+  <artifactId>producer-kafka</artifactId>
```

These should probably have their own version, right?
```suggestion
  <groupId>io.stargate.producer.kafka</groupId>
  <artifactId>producer-kafka</artifactId>
  <version>0.0.11-SNAPSHOT</version>
```

---

I've excluded kafka for now - it does not start in OSGi (at least in my env.)

---

@dimas-b can you please give @tomekl007 a quick write up of what you did so he can pull the producer stuff in during his day?

---

Hi @dimas-b, the whole producer work is on the feature branch (not `master`):
https://github.com/stargate/producer-extensions/pull/5
The work from this branch should be pulled in. If you need these changes to be on the master, feel free to merge the PR. The CI will not build because of:
https://github.com/stargate/producer-extensions/pull/6#issuecomment-700038823